addons/website/views/snippets/s_faq_horizontal.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_faq_horizontal" name="Topics">
+    <section class="s_faq_horizontal" data-oe-shape-data="{'shape':'web_editor/Origins/09_001','flip':[],'showOnMobile':false,'shapeAnimationSpeed':'0'}">
+        <div class="o_we_shape o_web_editor_Origins_09_001"/>
+        <div class="container">
+            <div class="row">
+                <div class="col-12 col-lg-6 offset-lg-3 pt48 pb48">
+                    <h2 style="text-align:center;">Topic Walkthrough</h2>
+                    <p style="text-align:center;">Learn how to quickly set up and start using our services with our step-by-step onboarding process.</p>
+                </div>
+            </div>
+            <div class="row s_nb_column_fixed">
+                <div data-name="Topic" class="s_faq_horizontal_entry col-12">
+                    <div class="row s_nb_column_fixed">
+                        <div class="col-lg-4 pb8">
+                            <div class="s_faq_horizontal_entry_title position-lg-sticky overflow-auto pb-lg-3" style="top: 16px">
+                                <h4>Getting Started</h4>
+                                <p class="o_small text-muted">Getting started with our product is a breeze, thanks to our well-structured and comprehensive onboarding process.</p>
+                            </div>
+                        </div>
+                        <div class="col-lg-7 offset-lg-1 pb32">
+                            <img src="/web/image/website.s_faq_horizontal_default_image_1" class="img img-fluid w-100 rounded" alt=""/>
```

Comment:
Unfortunately, if you want to take advantage of the WB to define the image width, you'll have to do in inline css 😬 
```suggestion
                            <img src="/web/image/website.s_faq_horizontal_default_image_1" class="img img-fluid rounded" alt="" style="width: 100% !important"/>
```
![Capture d’écran 2024-08-12 à 18 07 29](https://github.com/user-attachments/assets/3f68bbfb-504c-4f08-8767-00378cfec3da)